pkg/proto/v0/hello.pb.micro_test.go

```diff
@@ -0,0 +1,110 @@
+package proto_test
+
+import (
+	"context"
+	"log"
+	"testing"
+
+	"github.com/owncloud/ocis-hello/pkg/proto/v0"
+	"github.com/owncloud/ocis-pkg/v2/service/grpc"
+	"github.com/stretchr/testify/assert"
+
+	svc "github.com/owncloud/ocis-hello/pkg/service/v0"
+)
+
+var service = grpc.Service{}
+
+func init() {
+	service = grpc.NewService(
+		grpc.Namespace("com.owncloud.api"),
+		grpc.Name("hello"),
+		grpc.Address("localhost:9992"),
+	)
+
+	err := proto.RegisterHelloHandler(service.Server(), svc.NewService())
+	if err != nil {
+		log.Fatalf("could not register HelloHandler: %v", err)
+	}
+	service.Server().Start()
+}
+
+func TestCorrectService(t *testing.T) {
```

Comment:
you can save an allocation by doing "table driven tests" [like here](https://github.com/golang/go/wiki/TableDrivenTests)w

Reply:
The table is right below